CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+- Added uyuni-storage-setup-server package to Server deployment and
+  uyuni-storage-setup-proxy to Proxy deployment
```

Comment:
```suggestion
- Added uyuni-storage-setup-server package to server deployment and
  uyuni-storage-setup-proxy to proxy deployment in Installation and Upgrade
  Guide
```
FYI, we only capitalize server or proxy if "{productname}" or something similar precedes it.  It once was a Lana decision and I still support it ;)